.circleci/config.yml

```diff
@@ -25,7 +34,7 @@ jobs:
       - run: docker build -t $IMAGE_NAME:latest .
       - run: |
           echo "$DOCKERHUB_PASSWORD"| docker login -u $DOCKERHUB_USERNAME --password-stdin
-          docker push $IMAGE_NAME:latest
+          docker push $IMAGE_NAME:master
```

Comment:
I don't see where we create an image with the tag `master`, so I think this would fail.

Reply:
Oh, right, I need to update the build in that job.